src/test/codewhisperer/tracker/codewhispererTracker.test.ts

```diff
@@ -65,9 +65,11 @@ describe('codewhispererTracker', function () {
         })
 
         it('Should skip if telemetry is disabled', async function () {
+            globals.telemetry.telemetryEnabled = false
             const getTimeSpy = sinon.spy(Date.prototype, 'getTime')
             await CodeWhispererTracker.getTracker().flush()
             assert.ok(!getTimeSpy.called)
```

Comment:
Before the change to `codewhispererTracker.ts`, this test was accidentally passing.